src/device_interface.h

```diff
@@ -39,6 +39,10 @@ class DeviceInterface {
                               const std::vector<uint8_t>& payload,
                               bool expect_up_check,
                               std::vector<uint8_t>* response_cbor) const = 0;
+  // Sends a CTAPHID raw message to the device. Returns the response if received
+  // one.
+  virtual Status SendCtapHid(const std::vector<uint8_t>& payload,
+                             std::vector<uint8_t>* response) const = 0;
```

Comment:
The idea of `DeviceInterface` was that all functions be can implemented by each transport. So if you want a test function to send raw data, it should be called differently, like `ExchangeBytes`.

Ob "Exchange" vs "Send": Even though there is a case where we don't get a response, there's a return value called `response`. And just "Send" doesn't include the possibility of receiving.